allennlp/tests/data/tokenizers/pretrained_transformer_tokenizer_test.py

```diff
@@ -17,11 +50,32 @@ def test_splits_cased(self):
             ".",
             "[SEP]",
         ]
-        tokenizer = PretrainedTransformerTokenizer("bert-base-cased")
         tokens = [t.text for t in tokenizer.tokenize(sentence)]
         assert tokens == expected_tokens
 
-    def test_splits_uncased(self):
+        # sentence pair
```

Comment:
Similarly, this would be cleaner as a separate test case.

Reply:
Done